kythe/typescript/indexer.ts

```diff
@@ -463,6 +463,17 @@ class Visitor {
                 } else {
                   part = decl.name.text;
                 }
+                // Wrap literals in quotes, so that characters used in other
+                // signatures do not interfere with the signature creates by a
+                // literal. For instance, a literal
+                //   obj.prop
+                // may interefere with the signature of `prop` on an object
+                // `foo`. The literal receives a signature
```

Comment:
Wouldn't it interfere with an object `obj`, not `foo`?

Reply:
fixed